styleguide/_themes/global/scss/banners.scss

```diff
@@ -36,31 +36,147 @@ $banner-text-margin-medium: 10%;
 
 .banner-headline {
   @include heading($white);
-  font-size: 3em;
-  line-height: 1em;
+  font-size: 4rem;
+  line-height: 3.75rem;
   text-align: center;
 }
 
 
-.imacBanner-container {
-  background-color: $brand-primary;
+.imacBanner {
+  background-image: url('https://752f77aa107738c25d93-f083e9a6295a3f0714fa019ffdca65c3.ssl.cf1.rackcdn.com/bananers/backgrounds/o365-overview.jpg');
+  background-size: cover;
   color: $white;
+  margin-top: 50px;
+  overflow: hidden;
 
-  &.large {
+  &.small {
+    height: 250px;
+    padding-top: 50px;
+  }
+
+  &.medium {
     height: 300px;
-    padding-top: 30px;
+    padding-top: 50px;
+  }
+
+  &.large {
+    height: 350px;
+    padding-top: 50px;
   }
 }
 
+.imacBanner-container {
+  @include make-container;
+  @include fade(.25s, ease-in-out);
+  opacity: 0;
+}
+
+.wf-khand-n7-active {
+  .imacBanner-container {
+    opacity: 1;
+  }
+}
```

Comment:
So if we're going to do this, which I am in support of by the way, I think it makes sense to have all of these webfont overrides in their own file that are excluded from sass linting.  The main reason for that is this class names don't meet sass-lint guidelines, but since they're mandated by an external tool, I don't think we should have that causing issues.  
```
styleguide/_themes/global/scss/banners.scss
  74:2  warning  Class '.wf-khand-n7-active' should match regular expression /^([a-z0-9]+([A-Z][a-z0-9]+)*)(-{1}([a-z0-9]+([A-Z][a-z0-9]+)*)){0,2}$/  class-name-format
```

Alternatively, we can change our class name rules to allow 4 tokens, but I'm not sure we want to keep adjusting our rules everytime we come up with an exception.  That said, excluding from the rules may not be a good solution either.  This probably warrants further discussion with the team.